linode/sshkey/framework_datasource_schema.go

```diff
@@ -4,23 +4,25 @@ import (
 	"github.com/hashicorp/terraform-plugin-framework/datasource/schema"
 )
 
-var frameworkDatasourceSchema = schema.Schema{
-	Attributes: map[string]schema.Attribute{
-		"label": schema.StringAttribute{
-			Description: "The label of the Linode SSH Key.",
-			Required:    true,
-		},
-		"ssh_key": schema.StringAttribute{
-			Description: "The public SSH Key, which is used to authenticate to the root user of the Linodes you deploy.",
-			Computed:    true,
-		},
-		"created": schema.StringAttribute{
-			Description: "The date this key was added.",
-			Computed:    true,
-		},
-		"id": schema.StringAttribute{
-			Description: "A unique identifier for this datasource.",
-			Optional:    true,
-		},
+var SSHKeyAttributes = map[string]schema.Attribute{
+	"label": schema.StringAttribute{
+		Description: "The label of the Linode SSH Key.",
+		Required:    true,
+	},
+	"ssh_key": schema.StringAttribute{
+		Description: "The public SSH Key, which is used to authenticate to the root user of the Linodes you deploy.",
+		Computed:    true,
+	},
+	"created": schema.StringAttribute{
+		Description: "The date this key was added.",
+		Computed:    true,
+	},
```

Comment:
This field might be able to use the `RFC3339TimeStringType` CustomType: https://github.com/linode/terraform-provider-linode/blob/dev/linode/firewalls/framework_schema.go#L115

Reply:
Oh right, I'll change that now.